src/dataset.py

```diff
@@ -46,6 +50,8 @@ def __getitem__(self, i: int):
         for bi in batch_indices:
             img_name = Path(urlparse(self.images[bi]).path).name
             img_path = self.dataset_path / img_name
+            print(f'img_name={img_name} img_path={img_path} url={self.images[bi]}')
+            print(f'labels[bi]={self.labels[bi]}')
```

Comment:
This adds much noise, mb lets do
```suggestion
            logging.debug(f'img_name={img_name} img_path={img_path} url={self.images[bi]}')
            logging.debug(f'labels[bi]={self.labels[bi]}')
```